src/components/BButton/BButton.vue

```diff
@@ -81,5 +81,8 @@ button {
   background-color: #00AAED;
   border-color: #00AAED;
   color: #FFFFFF;
+  margin-bottom: 1.25rem;
+  padding-left: 0.9375rem;
+  padding-right: 0.9375rem;
 }
```

Comment:
There is pushback against this.  You should discuss with @eleanormal, Ryan and @cochoa89 and see if we need to remove this code.  Its a bit out of scope compared to the other work though.  I think it shoiuld be at least in its own PR.